package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "http-mitm-proxy",
-  "version": "1.1.0",
+  "name": "@mablhq/http-mitm-proxy",
+  "version": "1.1.0-mabl",
```

Comment:
Since we're changing the package name to have a `@mablhq/` prefix, I don't think we need the `-mabl` suffix on the version number